### ISSUE TITLE
Frontoffice modifications templates

### ISSUE THREAD

**Issue by AudreyRemy** [GeotrekCE/Geotrek-rando-v3#608]
Bonjour
Est-il possible de modifier les templates utilisés pour l'affichage en frontoffice au-delà de la simple personnalisation css ? Il s'agirait de modifier les templates de listes et de détails modification de la structure HTML. 
Existe-t-il un langage de templating (style smarty) permettant d'appeler des objets du backoffice ?
Existe-t-il un système de surcharge des templates existants ?
Nous avons pris note du fait qu'il est possible de modifier la page home en insérant deux fichiers HTML homeBottom et homeTop.
Merci d'avance.
Bonne journée

**Comment by camillemonchicourt** [comment]
Actuellement, on peut seulement surcoucher le CSS.
Il a été évoqué le fait de mieux identifier chaque bloc pour en faciliter la customisation CSS, même si un premier travail a été fait à ce niveau (https://github.com/GeotrekCE/Geotrek-rando-v3/issues/543).

Il a aussi été évoqué l'idée d'ajouter plus de zones où on peut inclure son propre HTML et javascript, pas seulement sur la HOME (https://github.com/GeotrekCE/Geotrek-rando-v3/issues/588).

Mais pour modifier la structure HTML des pages, rien de prévu actuellement.

Peux-tu préciser ce que tu souhaiterais faire et à quel niveau ? Car il y a potentiellement différentes pistes d'évolution selon le besoin.

**Comment by AudreyRemy** [comment]
Merci.
Il y a différentes demandes comme afficher plusieurs photos sur un itinéraires (sur la fiche détail il n'y en a plus qu'une alors que sur la liste des itinéraires il apparait bien le carrousel avec les différentes photos), ajouter un système de pop-up sur certains itinéraires, modifier les logos en fonctions des structures gestionnaires de l'itinéraire, ...

**Comment by camillemonchicourt** [comment]
Non, on a bien plusieurs photos avec un caroussel sur les fiches détails des randos : https://rando.ecrins-parcnational.fr/trek/920686-Aiguilles-de-Chabrieres

Les autres évolutions que tu évoques ne sont pas des customisations de template HTML, mais bien des évolutions globales à apporter à l'outil Geotrek-rando-v3 pour tous.
Donc ce seraient plutôt des développements d'évolutions à faire et à contribuer dans l'outil.

D'autant plus si vous voulez des logos par structures, alors il faut ajouter ça dans les données, dans l'API, etc... Donc c'est pas juste du HTML.

**Comment by babastienne** [comment]
Plusieurs idées intéressantes dans ce ticket. Ca vaudrait le coup d'en faire plusieurs tickets séparés par fonctionnalité ?

-----

Avoir un logo par structure gestionnaire est un sujet qu'on ma remonté lors d'un échange ce matin. Ca serait intéressant d'un point de vue "esthétisme" du site mais surtout pour permettre de valoriser le travail effectué par les structures au quotidien.

L'un des problèmes remonté dernièrement par des parcs est le suivant : difficulté à motiver les territoires partenaires à saisir de la donnée dans Geotrek car ils ont l'impression de contribuer aux données du Parc sans que cela leur profite réellement (en terme de visibilité). Ajouter sur les fiches des objets un logo avec la structure ayant créé l'objet ainsi que peut-être un lien redirigeant sur un site de la structure (ou sur une vue de GTR filtrée pour afficher uniquement les objets créés par la structure) permettrai de mettre plus en avant l'aspect communautaire de Geotrek. On m'a parlé récemment de territoires (OT, CC) qui n'avaient pas ce "réflexe Geotrek", or c'est en trouvant des petites features comme celles-ci qu'on pourra réussir à relancer un attrait des territoire pour Geotrek, ce qui in fine permettra de continuer à dynamiser la communauté.

**Comment by camillemonchicourt** [comment]
Par ailleurs depuis la version 2.80 de Geotrek-admin, il est aussi possible d'avoir un logo dans le PDF différent selon le portail.
Ou même d'avoir des éléments différents dans le PDF selon la structure qui a créé l'objet.

Je ferme le ticket si il n'y a pas d'autres besoins, ou à préciser dans des tickets dédiés ?

**Comment by AudreyRemy** [comment]
Merci Camille on utilise la version 2.75 donc je note cette info qui nous sera utile très prochainement.
